Example/StripeConnectExample/StripeConnectExample/API/API.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  API.swift
+//  StripeConnect Example
+//
+//  Created by Chris Mays on 8/23/24.
+//
+
+import Foundation
+
+struct API {
+    static func appInfo(baseURL: String = AppSettings.shared.selectedServerBaseURL) async -> Result<AppInfo, APIError> {
+        await apiRequest(path: "app_info", baseURL: baseURL)
+    }
+    
+    static func accountSession(merchantId: String? = nil, baseURL: String = AppSettings.shared.selectedServerBaseURL) async -> Result<AccountSessionResponse, APIError> {
+        await apiRequest(path: "account_session", 
+                         method: "POST",
+                         headers: merchantId.map { ["account": $0] } ?? [:],
+                         baseURL: baseURL)
+    }
+    
+    static func apiRequest<Response: Codable>(path: String,
+                                              method: String = "GET",
+                                              headers: [String: String] = [:],
+                                              baseURL: String = AppSettings.shared.selectedServerBaseURL) async -> Result<Response, APIError> {
```

Comment:
The other thing that I like to avoid here is wrapping everything in a do/catch. I feel like that can sometimes clutter the code. What do you think?

Reply:
>  Luckily we will get typed throws when we move to Swift 6

🥳

> The other thing that I like to avoid here is wrapping everything in a do/catch

I'm not strongly opinionated, so I think it's reasonable to keep this is a result. 

My personal opinion is that results force you to use switch statements, which isn't any cleaner than a try/catch. But throwing errors can potentially mean less logic handling because you can often defer it up the stack and consolidate the error handling in one place. But it's ultimately a personal preference 😄